docs/jwst/resample/arguments.rst

```diff
@@ -55,10 +55,14 @@ image.
   The value to assign to output pixels that have zero weight or do not
   receive any flux from any input pixels during drizzling.
 
-``--weight_type`` (str, default='exptime')
-  The weighting type for each input image. If `weight_type=exptime`,
-  the scaling value will be set equal to the exposure time found in
-  the image header.
+``--weight_type`` (str, default='ivm')
+  The weighting type for each input image.
+  If `weight_type=ivm` (the default), the scaling value
+  will be determined per-pixel using the inverse of the read noise
+  (VAR_RNOISE) array stored in each input image. If the VAR_RNOISE array does
```

Comment:
So not really an inverse variance (which would be readnoise^2 + Poisson Noise^2).  Or is the code correct?

Reply:
Oops - I see it's VAR_RNOISE which is correct of course.  Still, no Poisson contribution?

Reply:
No, that was the agreed upon method within the Cal WG, to use readnoise only. And the code does in fact set weight = var_rnoise**-1.

Reply:
Maybe I just missed the discussion where using only the VAR_RNOISE in the weight was justified

Reply:
Looks like I did miss it :).  Ignore me